README.md

```diff
@@ -41,7 +41,7 @@ docker-compose down
 
 which kills your containers but keeps your iamges. Be sure to use `docker-compose --help` for any additional help or other options.
 
-If the code has been changed, you need not rebuild using `docker-compose up -d --build` for backend or mongo but as of this moment and vue file changes will need to run the command.
+If the code has been changed, rebuild using `docker-compose up -d --build` if you have made changes to the frontend files. For backend or mongo containers they are reloaded live and any changed made on your system will be automatically refelected in the container and recompiled accoridngly.
```

Comment:
This is much better 😄 .... except for the spelling: changed => change, refelected => reflected and accoridngly => accordingly.

Reply:
English is not my strongest subject. Will fix now